### PR TITLE
chore(backend): Visualization: Using the correct Tensorflow image to prevent build time-outs

### PR DESCRIPTION
**Pull request by Ark-kun**
Decrease failures like the following: https://00e9e64bac96f3b5ceb1c3cfb1005d0a00d4d7f2cd03a990dc-apidata.googleusercontent.com/download/storage/v1/b/363997316495.cloudbuild-logs.googleusercontent.com/o/log-4b63e78a-ea23-4c52-b3f8-adcfa52f7dce.txt?qk=AD5uMEtFzOOoFfudNzxaUuDUFdIaxG_zPSxzo-bRUvvcNMZcfl4hOOouQsL6l6WObtQzXTlxCdNKGYS9eD1oRPD6QkBD7Tb5eTO2s2LECqJUkBMAiHyiaJYqyWHbNvKk6l3l5wjGrQx2ToBQkBTCJhNO_lttWaQUvTDN1acZU85s1K7X51e3Z7sB_hBMDKdHdjZPlNJKaHaQvUshQJRlE_4HMA40sEhECxMWb8xncijYL1Wijnppj1Y6f9ANFcqR2DsNqeC-fLVqPYpegj9idSVBW_z23iRZRCjzCOXzk-LLkkbe-O1XK_NCIeaUrzoDNll8hiI2tJ4Yy_ozVYXj4UNtMV4sxAR72i6w7dy2u-Q1U6I1ocwSCSH84UV0LPzPmo3c9w6vacjJVLm4DnQNBDc6RfNgojutNGXL8_hn8TXOVgXKMRs4SbbLr216QpQlXn9McG84GOsWM-V1ayyS5aUpW-4DubF3-aeGNKgyvMwnTBqVDTHoi1NflKXxRCDDAW1i43q4hbeiq4LJXRl_5OqHl1PEC8gAGgLKNDy1jXMp3NM6cJqF3VpPzNEgS_y3I1DqjyFavkqvFxYMM8FX6cOKf-rBteF8TMzu3kAlGAc-E4QmQSSP_ygjb5ISdjkQ7bwHAit0BmeBej7ldsPAV8web8xIu3akvMlH0ZFuX-m807cYdJZU26-OlPwafVS5J_iyg--GNgQTVcKydAcjf590pmUuRLONsic5rEg67zBw_YdWR5bQoDRr8XSyHsg8EjN7aIjXudE06VN_eQOCxG940MH-zjJKhO5tIZTdpJT4-mcAw8x7Pg4H3Vihx-MVhSWGUfRJIJXASNgYEBJ5DEH1wN41zadYnBsc9r6OuPwfI7Z96zhSbghFz60KnRZbIuMVXow8yTlP&isca=1

### CODE REVIEW

**Comment by Ark-kun** [comment]
```
Step #4 - "visualization-server": Some libraries are part of third_party_licenses.csv, but not installed.
Step #4 - "visualization-server": Please remove them from third_party_licenses.csv:
Step #4 - "visualization-server": Keras-Applications astor
Step #4 - "visualization-server": The command '/bin/sh -c ./license.sh third_party_licenses.csv /usr/licenses' returned a non-zero code: 1
```

**Comment by Ark-kun** [comment]
> Some libraries are part of third_party_licenses.csv, but not installed.

Fixed in https://github.com/kubeflow/pipelines/pull/4355

File: backend/Dockerfile.visualization [hunk 1]
```diff
@@ -17,7 +17,7 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-FROM tensorflow/tensorflow:2.1.0-py3
+FROM tensorflow/tensorflow:2.2.0
```
Comment (Bobgy) [inline backend/Dockerfile.visualization:20]: just realized you should also update: https://github.com/kubeflow/pipelines/blob/0b096de63faaf06c38446e0b053b87f7c51f3925/backend/src/apiserver/visualization/update_requirements.sh#L4

and Can you add a comment here reminding people about that

Comment (Ark-kun) [reply backend/Dockerfile.visualization:20]: Done.
Although it's not strictly required that these images are in sync. It just makes things faster.

**Pull request merged**